keyboards/planck/planck.h

```diff
@@ -117,7 +117,7 @@
     { k06, k07, k08, k09, k0a, k0b }, \
     { k16, k17, k18, k19, k1a, k1b }, \
     { k26, k27, k28, k29, k2a, k2b }, \
-    { k36, k37, k38, k33, k34, k35 } \
+    { k36, k36, k38, k33, k34, k34 } \
```

Comment:
This isn't the right way to do this. 

```suggestion
    { KC_NO, k37, k38, k33, KC_NO, k35 } \
```

And then change the k34 to k35, and the k36 to k37.

Reply:
Compiling with that I get errors saying k37 and k35 are both undeclared.

Reply:
Replacing those with 36 and 34 did it. Now the two 1u away from the center have no effect, but both the centered and innermost pins under where a 2u key would go are still active.
```suggestion
    { KC_NO, k36, k38, k33, KC_NO, k34 } \
```